client/checkout/classic/upe-deferred-intent-creation/payment-processing.js

```diff
@@ -172,13 +184,66 @@ export async function mountStripePaymentElement( api, domElement ) {
 		showErrorCheckout( error.message );
 		return;
 	}
+
+	/*
+	 * Trigger this event to ensure the tokenization-form.js init
+	 * is executed.
+	 *
+	 * This script handles the radio input interaction when toggling
+	 * between the user's saved card / entering new card details.
+	 *
+	 * Ref: https://github.com/woocommerce/woocommerce/blob/2429498/assets/js/frontend/tokenization-form.js#L109
+	 */
+	const event = new Event( 'wc-credit-card-form-init' );
+	document.body.dispatchEvent( event );
+
 	const paymentMethodType = domElement.dataset.paymentMethodType;
 	const upeElement =
 		gatewayUPEComponents[ paymentMethodType ].upeElement ||
 		( await createStripePaymentElement( api, paymentMethodType ) );
 	upeElement.mount( domElement );
 }
 
+export const createAndConfirmSetupIntent = ( $form, { id }, api ) => {
+	return api.setupIntent( id ).then( function ( confirmedSetupIntent ) {
+		appendSetupIntentToForm( $form, confirmedSetupIntent );
+		return confirmedSetupIntent;
+	} );
+};
+
+/**
+ * Saves the payment method ID in a hidden input, and re-submits the form.
+ *
+ * @param {Object} $form         The jQuery object for the form.
+ * @param {Object} paymentMethod Payment method object.
+ */
+export const handleOrderPayment = ( $form, { id } ) => {
```

Comment:
```suggestion

 * @param {Object} paymentMethod Payment method object.
 */
export const handleOrderPayment = ( { id } ) => {
```
First I thought this `$form` variable was unused and the parameter could be removed, but I see now that to follow [the callback signature for the `additionalActionsHandler`](https://github.com/Automattic/woocommerce-payments/blob/feature/deferred-upe-my-account-page/client/checkout/classic/upe-deferred-intent-creation/payment-processing.js#L287-L291), we need it after all...

...However, if you swapped the order of the parameters, so that the `paymentMethod` is used first, `$form` second, and `api` last, you could remove this unused parameter after all. `paymentMethod` is the only parameter required by both callback functions--`handleOrderPayment` & `createAndConfirmSetupIntent`--whereas the other parameters are technically not required, since they are only required by the latter. 

It's a minor complaint, but I think it's marginally cleaner, so let me know what you think!